01/ht/gendos379/03/script.js

```diff
@@ -0,0 +1,78 @@
+window.onload = function() {
+    var selectIds = {'rowsNumber': 100, 'colNumber': 20}
+    for (i = 0; i < Object.keys(selectIds).length; i++) {
+        var ddl = document.getElementById(Object.keys(selectIds)[i]);
+        for (var j = 1; j <= selectIds[Object.keys(selectIds)[i]]; j++) {
+            var option = new Option;
+            option.text = j;
+            option.value = j;
+            ddl.options[j] = option;
+        }
+    }
+};
+
+var createTableButton = document.getElementById('create-table');
+createTableButton.addEventListener('click', function() {
+    if (tableIsVaild()) {
+        createTable();
+    } else {
+        document.getElementById('table').innerHTML = 'Wrong parameters!';
+    }
+});
+
+function enableCaption() {
+    var captionCheck = document.getElementById('caption-check')
+    var input = document.getElementById('caption-name'); 
+    if (captionCheck.checked) {
+        input.disabled = false; 
+        input.focus(); 
+    } else {
+        input.disabled = true;
+    }
+};
+
+function tableIsVaild() {
+    if (document.getElementById('rowsNumber').value > 0 && document.getElementById('colNumber').value > 0) {

```

Comment:
just `return document.getElementById('rowsNumber').value > 0 && document.getElementById('colNumber').value > 0`  --- the same result